tools/devtool

```diff
@@ -631,6 +631,17 @@ update_python_package_version() {
     docker rm -f "$dummy_container_name"
 }
 
+create_dir() {
```

Comment:
Do we actually need to add this function? AFAIK we run the script with `set -e`.

Reply:
This piece of code is used in multiple places in devtool: like [here](https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L285) or [here](https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L1546). My thinking was to get rid of this duplicate code so I will also replace those occurrences.  I do not see any `set -e` in devtool...
i am thinking of posting an update with the deduplication of `mkdir -p ...`. What do you think?

Reply:
I updated the code, let me know what you think